### PR TITLE
Update Connect server to v1.1.0

### PR DESCRIPTION
**Pull request by florisvdg**

### CODE REVIEW

**Comment by jpcoenen** [comment]
We also have to package the Chart, right? We have `make release/prepare` to do this.

**Comment by florisvdg** [comment]
Yes, but this is not a release (prep) PR. Just a Docker image version bump.

File: connect/values.yaml [hunk 1]
```diff
@@ -16,7 +16,7 @@ connect:
     type: emptyDir
     values: {}
   imagePullPolicy: IfNotPresent
-  version: "1.0.0"
+  version:
```
Comment (thatguygriff) [inline connect/values.yaml:19]: Do we not need the interpolated `.Chart.AppVersion` here to match what the Readme says?

Comment (florisvdg) [reply connect/values.yaml:19]: Yeah, that's cleaner indeed.

**Review by thatguygriff — APPROVED** [review]

**Pull request merged**